boards/px4/fmu-v6xrt/nuttx-config/nsh/defconfig

```diff
@@ -185,7 +185,6 @@ CONFIG_LPUART8_TXDMA=y
 CONFIG_MEMSET_64BIT=y
 CONFIG_MEMSET_OPTSPEED=y
 CONFIG_MMCSD=y
-CONFIG_MMCSD_MULTIBLOCK_LIMIT=1
```

Comment:
We need to wait on this change until https://github.com/PX4/NuttX/pull/305 get's merged in Upstream and PX4.

Reply:
Indeed but right now `sd_bench -v -U` would fail if that PR isn't, I pulled these changes in just for verification.